rest_framework/filters.py

```diff
@@ -188,4 +188,7 @@ def filter_queryset(self, request, queryset, view):
             'model_name': get_model_name(model_cls)
         }
         permission = self.perm_format % kwargs
-        return guardian.shortcuts.get_objects_for_user(user, permission, queryset)
+        if guardian.VERSION >= (1, 3):
+            # Maintain behavior compatibility with versions prior to 1.3
+            extra = {'accept_global_perms': False}
+        return guardian.shortcuts.get_objects_for_user(user, permission, queryset, **extra)
```

Comment:
Noticed that this will break for version prior to 1.3 (extra won't be defined)
Resolved in master just now but your PR would also need to be updated too.